content/terraform/v1.14.x (alpha)/docs/cli/commands/query.mdx

```diff
@@ -0,0 +1,86 @@
+description: |-
+ Add the `locals` block to your `.tfquery.hcl` files define values to reuse within the local Terraform module. 
+---
+
+# `terraform query` command
+
+The `terraform query` command queries existing infrastructure for unmanaged resources according to the `tfquery.hcl` file so that you can import them into your Terraform workspace.
+
+## Usage
+
+```shell-session
+$ terraform query [flags] [options]
+```
+
+## Description
+
+When you run the `terraform query` command, Terraform searches the current configuration directory for `tfquery.hcl` files so that it can query remote infrastructure for resources that match the defined `list` blocks. Terraform prints the results to the terminal. You also add the `-generate-config-out` flag to generate configuration that you can use to import the resources. Refer to [Import resources in bulk](/terraform/language/import/bulk) for more information.
+
+## Flags
+
+- `-var '<INPUT-VARIABLE>=<VALUE>'`: Specifies a key-value pair that sets an input variable defined in the `tfquery.hcl` configuration. You can use this flag more than once to set additional input variables.   
+    - Data type: String. 
+    - Example: [Set input variables](#set-input-variabls)
+
+- `-var-file=<FILENAME>`: Specifies a file containing input variable values, in addition to the default `terraform.tfvars` and `*.auto.tfvars` files.  
+    - Data type: Filename 
+    - Example: [Set input variables](#set-input-variabls)
+
+## Options
+
+- `-generate-config-out=<PATH/TO/OUTPUT>`: Instructs Terraform to generate `import` and `resource` blocks for results so that you can import them. Terraform writes the configuration to a new file. The file must not already exist. When used with the `-json` option, Terraform generates the configuration as part of the JSON output instead of writing it to a file. 
+    - Data type: Path 
+    - Example: [Generate import configuration](#generate-import-configuration)
+
+- `-json`: Instructs Terraform to print results to the console in JSON format. 
+     - Data type: Boolean
+     - Example: [Generate import configuration](#generate-import-configuration)
+
+- `-no-color`: Terraform prints monochrome results. 
+     - Data type: Boolean
+## Examples
+
+The following examples show how to use the command for common use cases.
+
+### Set input variables
+
+The following command passes sets the value of the `env` input variable to `prod`:
+
+```shell-session
+$ terraform query -var 'env=prod'
+```
+
+Corresponding `tfquery.hcl` configuration:
+
+```hcl
+list "aws_instance" "instances" {
+  provider = aws
+}
+
+variable "env" {
+  type = string
+  default = "test"
+}
+```
+
+The following command loads variable values defined in `my-vars.tfvars`:
+
+```shell-session
+$ terraform query -var-file=my-vars.tfvars
+```
+
+### Generate import configuration
+
+The following command generates `import` and `resource` blocks for the query results to a file in the `/to-import` directory:
```

Comment:
```suggestion
The following command generates `import` and `resource` blocks for the query results to a file in the `to-import` directory:
```

Technically that leading slash would mean it creates it in the `to-import` directory in the root of the file system rather than as a relative path.